_posts/2010-01-04-testing-km.md

```diff
@@ -6,78 +6,86 @@ tags: [developer_portal]
 summary: Learn how to test Kissmetrics and keep the test data separate from your "real" data.
 permalink: /getting-started/testing-km/index.html
 ---
+#How to Verify that Kissmetrics is Working
+
+The first time you set up Kissmetrics, you can create a Test Site and use Kissmetrics Live to verify that Kissmetrics is working. The Test Site allows you to create events without harming the real data you collect from your Production Site. 
+
+In addition, if you create a new product, you will receive an additional API key so that you can reserve it for testing purposes.
+
 * Table of Contents
 {:toc}
 * * *
 
-When you are starting out, we highly recommended that you set up a separate Test Site as a sandbox. That way, any events you create while testing will not interfere with the real data you collect from your Production site.
+==============================================================================================================================
```

Comment:
If you want a horizontal line, it's already in place due to the 3 asterisks under `{:toc}`